### PR TITLE
Jmpi 66 fix import endpoint max file size

### PR DESCRIPTION
**Pull request by IlyesBenAmara**
- This PR adds a max file size for the user to import to JeMPI

### CODE REVIEW

File: docker/conf/env/conf-env-1-pc.template [hunk 1]
```diff
@@ -76,3 +76,4 @@ export KC_JEMPI_CLIENT_ID=${KC_JEMPI_CLIENT_ID}
 export KC_JEMPI_CLIENT_SECRET=${KC_JEMPI_CLIENT_SECRET}
 export KC_JEMPI_ROOT_URL=${KC_JEMPI_ROOT_URL}
 export JEMPI_SESSION_SECRET=${JEMPI_SESSION_SECRET}
+export FILE_IMPORT_MAX_SIZE=${FILE_IMPORT_MAX_SIZE}
```
Comment (marrouchi) [inline docker/conf/env/conf-env-1-pc.template:79]:
```suggestion
export JEMPI_FILE_IMPORT_MAX_SIZE_BYTE=${JEMPI_FILE_IMPORT_MAX_SIZE_BYTE}
```
Let's prefix them all with "JEMPI_" et suffix them with "BYTE

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Review by marrouchi — APPROVED** [review]

**Review by marrouchi — COMMENTED** [review]
Might be good to add something about this endpoint in the documentation (documentation/api.md).

**Comment by IlyesBenAmara** [comment]
@marrouchi Will do :technologist: :+1:

**Review by MatthewErispe — APPROVED** [review]
LGTM

**Pull request merged**